hack/lib.bash

```
@@ -0,0 +1,364 @@
+timeout="10m"
+
+log() { echo ">>> $@"; }
```

Shellcheck (0.11.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)